packages/components/src/styles/components/tabs.scss

```diff
@@ -77,6 +78,7 @@
   align-items: center;
   padding: 0;
   color: inherit;
+  font-weight: var(--token-typography-font-weight-medium);
```

Comment:
[blocker] the class `hds-typography-body-200` comes with a `font-family`, but we haven't added it to the CSS styles when we moved the typography here, we have to add it (if you inspect the button in the DevTools, you will see it's "Arial")

Reply:
Where/how should I add the font-family style?

Reply:
where the `hds-typography-body-200` was applied before, that is at `.hds-tabs__tab-button` level